app/controllers/votes_controller.rb

```diff
@@ -0,0 +1,16 @@
+class VotesController < ApplicationController
+  def create
+    @review = Review.find(params[:review_id])
+    @vote = Vote.find_or_initialize_by(user: current_user, review: @review)
+    @vote.value = params[:value]
+    @vote.save
+    render json: @review.tally
+  end
+
+  private
+
+  def vote_params
+    params.require(:vote).permit(:value)
+    .merge(user_id: current_user.id, review_id: params[:review_id])
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Use 2 (not 0) spaces for indenting an expression spanning multiple lines.